test/constructors.jl

```diff
@@ -34,39 +34,68 @@ const ≅ = isequal
                             CategoricalVector{Union{Float64, Missing}}(ones(3))])
     @test df == DataFrame([CategoricalVector{Union{Float64, Missing}}(zeros(3)),
                            CategoricalVector{Union{Float64, Missing}}(ones(3))], [:x1, :x2])
+    @test df == DataFrame([CategoricalVector{Union{Float64, Missing}}(zeros(3)),
+                           CategoricalVector{Union{Float64, Missing}}(ones(3))], ["x1", "x2"])
     @test df == DataFrame(Any[CategoricalVector{Union{Float64, Missing}}(zeros(3)),
                               CategoricalVector{Union{Float64, Missing}}(ones(3))])
     @test df == DataFrame(Any[CategoricalVector{Union{Float64, Missing}}(zeros(3)),
                               CategoricalVector{Union{Float64, Missing}}(ones(3))], [:x1, :x2])
+    @test df == DataFrame(Any[CategoricalVector{Union{Float64, Missing}}(zeros(3)),
+                              CategoricalVector{Union{Float64, Missing}}(ones(3))], ["x1", "x2"])
     @test df == DataFrame(AbstractVector[CategoricalVector{Union{Float64, Missing}}(zeros(3)),
                                          CategoricalVector{Union{Float64, Missing}}(ones(3))], [:x1, :x2])
+    @test df == DataFrame(AbstractVector[CategoricalVector{Union{Float64, Missing}}(zeros(3)),
+                                         CategoricalVector{Union{Float64, Missing}}(ones(3))], ["x1", "x2"])
     @test df == DataFrame((CategoricalVector{Union{Float64, Missing}}(zeros(3)),
                            CategoricalVector{Union{Float64, Missing}}(ones(3))))
     @test df == DataFrame((CategoricalVector{Union{Float64, Missing}}(zeros(3)),
                            CategoricalVector{Union{Float64, Missing}}(ones(3))), (:x1, :x2))
+    @test df == DataFrame((CategoricalVector{Union{Float64, Missing}}(zeros(3)),
+                           CategoricalVector{Union{Float64, Missing}}(ones(3))), (:x1, :x2))
+    @test df == DataFrame((CategoricalVector{Union{Float64, Missing}}(zeros(3)),
+                           CategoricalVector{Union{Float64, Missing}}(ones(3))), ("x1", "x2"))
```

Comment:
These are duplicated AFAICT.
```suggestion
```

Reply:
they are not duplicated as they test tuple interface (earlier tests check vector interface)

Reply:
Both seem to test tuples to me.

Reply:
I have rewritten this part of tests entirely to make them more readable. I hope now it will be more clear (and it is shorter).